### DISCUSSION TITLE
Error initialising app: [object Object]

### DISCUSSION THREAD

**Discussion by psmiratisu** [Budibase/budibase#7409] [Help] [answered]
I am not able to open my app and edit them. Cannot update them either. Error message is always "Error initialising app: [object Object]" "Error updating app: [object Object]" etc

Any idea why?

**Comment by czarnndz** [comment]
I have the same issue with all cloud applications

**Comment by shogunpurple** [comment, marked as answer]
Hi, we are currently looking into this one as a top priority to find the root cause. You should now be able to use your apps as normal in the meantime - can you try again and let me know if that's working?

**Reply by psmiratisu** [reply]
User can use the apps but developer not able to develop them. 

Thanks a lot! Please update us once it has been fixed!

**Reply by shogunpurple** [reply]
@OnziDreox are you still seeing the same issue when you try to open an app in the builder?

**Reply by psmiratisu** [reply]
oh i can see them now! thanks

**Reply by aria-blackford** [reply]
> @OnziDreox are you still seeing the same issue when you try to open an app in the builder?

I'm still having this same issue.

**Reply by shahmimajid** [reply]
Same goes for me. Still having this issue

**Comment by cardonasMind** [comment]
I have same error, even when I try to delete an app I get error "Can´t delete this app" (or something like that)

**Comment by cardonasMind** [comment]
![image](https://user-images.githubusercontent.com/56924778/186296310-b8220296-5584-4334-bcf7-5121c313e004.png)

**Comment by johnathanmulhall** [comment]
I, too, am having the same issue.

**Comment by LinhQuangHua** [comment]
Hi everyone,

Please try clearing browser cache then login again, I just tried and it worked.

**Reply by shahmimajid** [reply]
cool , thank for the tips. Now I'm able to view my app after using different browser
